src/components/utils/popover.ts

```diff
@@ -255,6 +302,8 @@ export default class Popover {
    */
   private createItem(item: PopoverItem): HTMLElement {
     const el = Dom.make('div', Popover.CSS.item);
+
+    el.setAttribute('data-tool', item.name);
```

Comment:
```suggestion
    el.setAttribute('data-item-name', item.name);
```
A popover shouldn't know what does the `tool` mean